solutions/3.js

```diff
@@ -0,0 +1,14 @@
+const multiple = (a,string,i=0)=>{
+  let str="";
+  if(i==a)
+  {
+   return str;
+  }
+  else
+  {
+    str=str+string;
+    console.log(str);
```

Comment:
please, no console.log in production code.

Reply:
ill check song

Reply:
resolved